### ISSUE TITLE
Latest patchwork missing argument warning for 'define' redefinitions.

### ISSUE THREAD

**Issue by krodyrobi** [antecedent/patchwork#75]
Minimal reproducible project [here](https://github.com/krodyrobi/patchwork-warning)
Only exhibited as far as I can see on php 5.6 and not php 7.1.

```
  "require-dev": {
    "phpunit/phpunit": "^5.7",
    "antecedent/patchwork": "2.1.*"
  }
```

You just need to issue a `docker-compose up`

```
1) SampleTest::test_define
Missing argument 4 for Patchwork\Redefinitions\define(), called in /app/tests/SampleTest.php on line 18 and defined
```

**Comment by antecedent** [comment]
Finally fixed in Patchwork 2.1.7. Apologies for the delay.

(Actually, just added an extra condition, as [it's a bug](https://3v4l.org/Ruc0o) in PHP reflection prior to 7.0.)

**Comment by krodyrobi** [comment]
Thank you!
